docs/index.rst

```diff
@@ -0,0 +1,15 @@
+Welcome to text_extensions_for_pandas's documentation! 
```

Comment:
Should this file be in git, or is it generated on demand?

Reply:
this file needs to be in git. its the landing page, and isn't generated automatically.